### DISCUSSION TITLE
Header center component growsover the header right

### DISCUSSION THREAD

**Discussion by Ferossgp** [software-mansion/react-native-screens#991] [Q&A] [answered]
Currently header center grows over the header right. I can see that on the initial render and during the transition it has the right width, then grows over the right size. 

I would expect it to have the same width as the title, and as in the initial render and during the transition.

If this is intentional, then maybe it is possible to an option to limit the width?

Expo Snack https://snack.expo.io/@ferossgp/trusting-cake

**Comment by tboba** [comment, marked as answer]
Hi @Ferossgp, thanks for reporting that! Sorry also for the late response 🙏 
The problem with positioning elements on the header is that views on which items are being placed are sized dynamically - there's no static width that indicates the maximum width of the elements on the header. 
We may modify it in the future, but for now it's not a priority for now.

Cheers!
